auto-value-gson/src/test/java/com/ryanharter/auto/value/gson/AutoValueGsonAdapterFactoryProcessorTest.java

```diff
@@ -45,6 +45,42 @@
             + "  }\n"
             + "  abstract String getName();\n"
             + "}");
+    // This is generated into a different package and not visible to the factory
```

Comment:
This could use a test for types that are visible but in a different package (which I think will currently fail, based on my other comments).